src/app/components/file-upload/file-upload.component.html

```diff
@@ -2,12 +2,17 @@
     <div class="child">
         <form [formGroup]="uploadForm" (ngSubmit)="onSubmit()">
             <div>
-                <input class="file-input" type="file" accept=".yaml,.json" name="swaggerFile"
+                <input #fileIn class="file-input" type="file" accept=".yml, .yaml,.json" name="swaggerFile"
```

Comment:
I am unfamiliar with this syntax. What does `#fileIn` accomplish here?

Reply:
I am guessing it is tied to the `errorMsg` div below?

Reply:
`#fileIn` is added to the input so I can read it in the component. Then `@ViewChild` in the ts component, it is used to access the input in the component. All in all, this is used to reset the file input text if swagger file fails to validate.